### ISSUE TITLE
Use case review: Accessibility and Personalization

### ISSUE THREAD

**Issue by rdeltour** [w3c/dpub-pwp-ucr#9]
> - 7. [Accessibility and Personalization](https://www.w3.org/dpub/IG/wiki/UseCase_Directory#Accessibility_and_Personalization)
>   - 7.1. WCAG 2.0 & Education Environment / Educational Publishing
>     - 7.1.1. Pre-School through Middle School
>   - 7.2. Specialized Subject Areas (e.g. Chemistry, Grammar, Poetry)
>   - 7.3. Allowing Versions
>   - 7.4. General Accessibility Use Cases (including Math)
>   - 7.5. Personalization
>   - 7.6. User Descriptions

Markus [said](https://www.w3.org/2016/01/11-dpub-minutes.html#item01) these may be lower priority. Most of these may be out of scope because they're not specific to PWP? In the A11y TF note?
## WCAG 2.0 & Education Environment
- ["Timed Tests and Activities"](https://www.w3.org/dpub/IG/wiki/Timed_Tests_UC) is about declaring/allowing additional time on timed activities and tests.  
  **Evaluation**: out of scope, not specific to PWP
- ["Mirroring Source Materials"](https://www.w3.org/dpub/IG/wiki/Mirror_Sources_UC) is about syncing an alternate accessible version with a lesser accessible source material.  
  **Evaluation**: out of scope, not specific to PWP
- ["Still View of Moving Content"](https://www.w3.org/dpub/IG/wiki/Still_View_Moving_Content) is about the ability to stop/pause timed content.  
  **Evaluation**: out of scope, not specific to PWP

There are other bullets but not associated to fully-described use cases.
## Specialized Subject Areas

One not-fully-described UC about the possibility for a publisher to provide samples of a larger pub.

**Evaluation**: this may not be directly in scope of this document, as it's probably more linked to work on metadata and identifiers?
## Allowing Versions
- ["Allow Different Math Pronunciations"](https://www.w3.org/dpub/IG/wiki/Allow_Different_Math_Pronunciations) is about configuring different TTS (of various quality levels) depending on preference or reader level.  
  **Evaluation**: out of scope
- ["Facilitate Post Production Crowdsourcing of SVG Descriptions"](https://www.w3.org/dpub/IG/wiki/SVGCrowdDesc) is about bolt-on post-publication image description.  
  **Evaluation**: is it any different from annotation UCs?
- UC about easily deliver a patched version of a buggy publication  
  **Evaluation**: not specific to a11y, the general delivery issue is discussed elsewhere.
- UC about "improving the accessibility of an interactive widget"  
  **Evaluation**: too sketchy, but probably not specific to PWP?
- UC about quickly identify the version of a PWP for tech support  
  **Evaluation**: not specific to a11y. locator/identifier discussed elsewhere.
- UC about "needing clear identification of the source of the content they receive and to be able to allow/disallow content depending on the source"  
  **Evaluation**: I'm not sure I grasp the essence. Out of scope?
## General Accessibility Use Cases

About a dozen UCs related to a11y of content, which IMO are out of scope for this document. I suggest the A11y TF review these, and identify which one they want included.

A couple UCs that may be in scope:
- A UC about publishers and content remixers who want to keep image descriptions and alternatives in the package  
  **Evaluation**: relevant packaging UC, but maybe nothing specific compared to other packaging UCs?
- A couple UC about a11y feature metadata for discoverability  
  **Evaluation**: relevant metadata UC, see also the EPUB 3.1 approach
- A UC about synchronized word-level highlighting and TTS, like EPUB media overlays  
  **Evaluation**: is "media overlays for PWP" in scope for this doc?
## Personalization
- ["Adjustable Fonts"](https://www.w3.org/dpub/IG/wiki/Adjustable_Fonts) is about customizing font size+family.  
  **Evaluation**: in scope? also discussed in the Styling UCs.
- ["Adjustable Line Height and Length"](https://www.w3.org/dpub/IG/wiki/Adjustable_Line_Height_and_Length)  
  **Evaluation**: same as above.
- ["Adjustable Text Alignment"](https://www.w3.org/dpub/IG/wiki/Adjustable_Text_Alignment)  
  **Evaluation**: same as above.
- ["Adjustable Background and Contrast"](https://www.w3.org/dpub/IG/wiki/Adjustable_Background_and_Contrast)  
  **Evaluation**: same as above.
- "Zoom Control" is empty...
- ["Rendition Control"](https://www.w3.org/dpub/IG/wiki/Rendition_Control) is about multiple renditions (e.g. fixed layout + reflowable) of a publication.  
  **Evaluation**: are multiple renditions in scope for PWP?
- ["Initial Collection of Existing Efforts"](https://www.w3.org/dpub/IG/wiki/Accessibility_and_Personalization) describes existing efforts and is about synchronization with other initiatives.  
  **Evaluation**: Not a UC. A11y TF to decide if/how this must be reflected in this doc.
## User Descriptions

 ["ReadingDisorder"](https://www.w3.org/dpub/IG/wiki/ReadingDisorder_UC#Accessibility), ["Dyslexia"](), and ["CerebralPalsy"]() are UC describing personas, spawning various a11y requirements.  
  **Evaluation**: the scenarios are good, but the specific relation to PWP is not clear IMO (how does it differ from the generic Web?). The a11y TF should review these and decide if/how to consolidate.

**Comment by TzviyaSiegman** [comment]
IndieUI will re-launch in WAI soon (timeline TBD). It will be very helpful to have use cases to turn over to them when they start, but this is not our first priority.

**Comment by deborahgu** [comment]
The accessibility TF did not do a great job of resolving  the relationship is between these use cases to the note that we have been creating; putting this on the agenda for our next meeting. (cc: @clapierre)

I will say that for our purposes we did not only focus on ideas that were unique to PWP (I'd argue that _nothing_ is unique to PWP except the spec for PWP), but on any idea which is an unsolved problem & which is extremely important to the industry. E.g. dealing with accessibility of timed tests is flat-out vital to digital publishing, regardless of who else it may be important for as well. I would argue that timed test accessibility is much less generic than the personalization points, which will likely apply to any web resource.

**Comment by deborahgu** [comment]
To be more precise, these use cases have been not reviewed at all by the accessibility TF. We should discuss on the TF:
1. 1. The difference between the purposes of the use cases and the purpose of the note.
2. How much of the use cases should be incorporated into the note's future work. 
3. Whether we are going to do a full editorial review of the accessibility use cases as part of the TF once we have a better understanding of their purpose.

cc @clapierre

**Comment by clapierre** [comment]
Agreed this is something we will discuss in the DPUB A11Y TF and figure out what we may need to add to our Note regarding use cases for Accessibility & Personalization.  I have added this to our agenda this week.
